python/ray/tune/tune.py

```diff
@@ -167,8 +173,8 @@ def run(run_or_experiment,
         verbose (int): 0, 1, or 2. Verbosity mode. 0 = silent,
             1 = only status updates, 2 = status and trial results.
         resume (str|bool): One of "LOCAL", "REMOTE", "PROMPT", or bool.
-            LOCAL/True restores the checkpoint from the local_checkpoint_dir.
-            REMOTE restores the checkpoint from remote_checkpoint_dir.
+            LOCAL/True restores the checkpoint from the local_dir.
```

Comment:
I think this is wrong right? It should be `local_dir/name`.

Reply:
Which is the `local_checkpoint_dir`.

Reply:
ah i see sorry, i think its just a little confusing because `local_checkpoint_dir` and `remote_checkpoint_dir` aren't exposed to the user right? so they don't really know what it is (afaik)